src/frontend/src/store/index.ts

```diff
@@ -86,11 +89,21 @@ export default new Vuex.Store({
         })
         .then((response) => {
           if(response.data.success) {
-            alert("Transaction was successful!");
-            dispatch("ACTION_FETCH_WALLET_AMOUNT");
-          } else if(response.data.err) {
-            alert("Transaction has failed.");
-          }
+            Vue.$toast.success('Transaction has sent!', {
```

Comment:
Can you add screenshots/gifs of this new stuff please, always nice to see new front end changes in their PRs